centreon/tests/e2e/features/Cloud-notifications/common.ts

```diff
@@ -180,11 +215,342 @@ const waitUntilLogFileChange = (): Cypress.Chainable => {
+      return null;
+    }
+
+    if (servicesFoundStepCount < maxSteps) {
+      cy.wait(stepWaitingTime);
+
+      return cy.wrap(null).then(() =>
+        checkServices({
+          acknowledged,
+          output,
+          status,
+          inDowntime,
+          statusType
+        })
+      );
+    }
+
+    throw new Error(
+      `The 1000 services are not monitored after ${pollingCheckTimeout}ms`
+    );
+  });
+};
+
+const initializeDataFiles = () => {
+  // Generate values for centreon_storage_services.txt
+  let centreonStorageServicesValues = '';
+  // The first service will got an id of 28
+  for (let i = 28; i < 1028; i++) {
+    const host_id = 15;
+    const description = `service_${i - 27}`;
+    const service_id = i;
+    const acknowledged = 0;
+    const acknowledgement_type = 0;
+    const action_url = '';
+    const active_checks = 0;
+    const check_attempt = 1;
+    const check_command = 'check_centreon_ping!3!200,20%!400,50%';
+    const check_freshness = 0;
+    const check_interval = 5;
+    const check_period = '24x7';
+    const check_type = 0;
+    const checked = 0;
+    const default_active_checks = 0;
+    const default_event_handler_enabled = 1;
+    const default_flap_detection = 1;
+    const default_notify = 0;
+    const default_passive_checks = 1;
+    const display_name = `service_${i - 27}`;
+    const enabled = 1;
+    const event_handler = '';
+    const event_handler_enabled = 1;
+    const execution_time = 0;
+    const first_notification_delay = 0;
+    const flap_detection = 1;
+    const flap_detection_on_critical = 1;
+    const flap_detection_on_ok = 1;
+    const flap_detection_on_unknown = 1;
+    const flap_detection_on_warning = 1;
+    const flapping = 0;
+    const freshness_threshold = 0;
+    const high_flap_threshold = 0;
+    const icon_image = '';
+    const icon_image_alt = '';
+    const last_hard_state = 0;
+    const last_update = 1710846876;
+    const latency = 0;
+    const low_flap_threshold = 0;
+    const max_check_attempts = 1;
+    const next_check = 0;
+    const no_more_notifications = 0;
+    const notification_interval = 0;
+    const notification_number = 0;
+    const notification_period = '24x7';
+    const notify = 0;
+    const notify_on_critical = 1;
+    const notify_on_downtime = 0;
+    const notify_on_flapping = 0;
+    const notify_on_recovery = 1;
+    const notify_on_unknown = 0;
+    const notify_on_warning = 1;
+    const obsess_over_service = 1;
+    const output = '';
+    const passive_checks = 1;
+    const percent_state_change = 0;
+    const perfdata = '';
+    const retain_nonstatus_information = 1;
+    const retain_status_information = 1;
+    const retry_interval = 1;
+    const scheduled_downtime_depth = 0;
+    const should_be_scheduled = 0;
+    const stalk_on_critical = 0;
+    const stalk_on_ok = 0;
+    const stalk_on_unknown = 0;
+    const stalk_on_warning = 0;
+    const state = 0;
+    const state_type = 1;
+    const volatile = 0;
```

Comment:
you don't need to declare all those constants if you only use those in `values` array

Reply:
Declaring those constants makes the code more comprehensive; otherwise, nobody would know which column each value refers to.

Reply:
IMHO it does not make it more comprehensive since the keys of `values` are exactly the same than the name of the constants